fix: 🐛 sd-audio a11y and other things (#2136)
solid-design-system/solid: .changeset/fine-planets-study.md

---
'@solid-design-system/components': patch
---

> fix: update aria-label in sd-audio for playback speed to include current speed value

Reviewer: ```suggestion
Improved `sd-audio` `aria-label` by including current speed value on playback speed button.
```
